properties/P000204.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: P000204
+name: Has a cut point
+refs:
+  - zb: "0684.54001"
+    name: General Topology (Engelking, 1989)
+---
+
+There exists a cut point $p \in X$.
+
+The point $p$ is called a *cut point* of $X$ if $X$ is connected and $X \backslash \{p\} = U \cup V$, for open, disjoint, and nonempty sets $U$ and $V$.
```

Comment:
```suggestion
A point $p$ is called a *cut point* of $X$ if $X$ is {P36} but $X \setminus \{p\}$ is not.
```
I think this is clearer? The latter part is just the definition of being disconnected.

Reply:
Yes, I agree. I was trying to be faithful to the text from Engelking, but I like this one more.

Reply:
Was the line deleted between 'Has a cut point' and the definition of cut point intentional? I was trying to following the style from https://topology.pi-base.org/properties/P000201